Translations/bg.po

```diff
@@ -1,11 +1,7 @@
-# Translation of DevilutionX to Bulgarian
-# Nikolay Popov <nikolai.popovz@gmail.com>, 2021.
-# @Krezzin, 2021.
-#
```

Comment:
Any reason you removed the credits?

Reply:
No, I overwrote the file. with a version I already have been working on, and forgot to include them. Will update it now to include them.